src/ToDo/Presentation/WelcomeViewModel.cs

```diff
@@ -19,7 +19,6 @@ public partial class WelcomeViewModel
 	public async ValueTask GetStarted(CancellationToken ct)
 	{
 		var user = await _authService.AuthenticateAsync(_dispatcher);
```

Comment:
If there is no changes for WelcomeViewModel @nickrandolph maybe we can just remove it for this PR